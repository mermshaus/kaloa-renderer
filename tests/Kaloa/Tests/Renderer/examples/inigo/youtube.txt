[youtube=dQw4w9WgXcQ][/youtube]

[youtube="dQw4w9WgXcQ"][/youtube]

[youtube id="dQw4w9WgXcQ"][/youtube]