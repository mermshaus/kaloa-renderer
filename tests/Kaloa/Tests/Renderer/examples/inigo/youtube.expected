<div class="videoWrapper">
  <iframe width="560" height="349" src="http://www.youtube.com/embed/dQw4w9WgXcQ" frameborder="0"></iframe>
</div>

<div class="videoWrapper">
  <iframe width="560" height="349" src="http://www.youtube.com/embed/dQw4w9WgXcQ" frameborder="0"></iframe>
</div>

<div class="videoWrapper">
  <iframe width="560" height="349" src="http://www.youtube.com/embed/dQw4w9WgXcQ" frameborder="0"></iframe>
</div>